test/suite/spring/jx_create_spring.go

```diff
@@ -35,7 +35,7 @@ var _ = Describe("create spring\n", func() {
 	Describe("Given valid parameters", func() {
 		Context("when running jx create spring", func() {
 			It("creates a spring application and promotes it to staging\n", func() {
-				args := []string{"project", "spring", "-b", "--org", T.GetGitOrganisation(), "--artifact", T.ApplicationName, "--name", T.ApplicationName, "-j", T.JavaVersion, "-d", "web", "-d", "actuator"}
+				args := []string{"project", "spring", "-b", "--org", T.GetGitOrganisation(), "--artifact", T.ApplicationName, "--name", T.ApplicationName, "-j", T.JavaVersion, "-d", "web", "-d", "actuator", "--type", "maven-project"}
```

Comment:
Would be good to not hard code this, but probably going to see if this fixes the issue with e2e tests ... and then add an option to T ...